worldedit-core/src/main/java/com/sk89q/worldedit/command/ToolCommands.java

```diff
@@ -191,6 +193,19 @@ public void tree(Player player, LocalSession session,
         setTool(player, session, new TreePlanter(type), "worldedit.tool.tree.equip");
     }
 
+    @Command(
+        name = "stacker",
+        desc = "Block stacker tool"
+    )
+    @CommandPermissions("worldedit.tool.stack")
+    public void stacker(Player player, LocalSession session,
+                      @Arg(desc = "The max range of the stack", def = "10")
+                        int range,
+                      @Arg(desc = "The mask to replace until", def = "!#existing")
```

Comment:
What does "The mask to replace until" actually mean? Does it stop stacking when the mask becomes `false`? If not, this wording is confusing.